Overseer/Helpers/DataContext.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using OneClickDesktop.Overseer.Entities;
+
+namespace OneClickDesktop.Overseer.Helpers
+{
+    public class DataContext : DbContext
+    {
+        public DbSet<User> Users { get; set; }
+
+        private readonly IConfiguration Configuration;
+
+        public DataContext(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder options)
+        {
+            // in memory database used for simplicity, change to a real db for production applications
+            options.UseInMemoryDatabase("TestDb");
+        }
+    }
+}
```

Comment:
Czy jest nam to potrzebne?

Reply:
TO raczej kwestia elastycznosci przy przerabianiu na LDAPa